invenio_communities/assets/semantic-ui/js/invenio_communities/members/searchBarWithFilters.js

```diff
@@ -84,13 +117,17 @@ class DropdownSort extends Component {
 }
 
 DropdownSort.propTypes = {
-  updateQueryState: PropTypes.object.isRequired,
-  currentQueryState: PropTypes.func.isRequired,
+  updateQueryState: PropTypes.func.isRequired,
+  currentQueryState: PropTypes.object.isRequired,
   filterKey: PropTypes.string.isRequired,
   filterValues: PropTypes.array.isRequired,
   filterLabel: PropTypes.string.isRequired,
 };
 
+DropdownSort.defaultProps = {
+  filterLabel: "",
+};
+
```

Comment:
could we place the component classes in different files? it would be more compact to read the code. Maybe an additional folder would help to keep them in one place

Reply:
Yes, good point!